oemof/solph/network.py

```diff
@@ -117,6 +117,23 @@ class Flow(on.Edge):
         :class:`Flow <oemof.solph.blocks.Flow>`.
         Note: at the moment this does not work if the investment attribute is
         set .
+    schedule : numeric (sequence or scalar)
+        Schedule for the flow. Flow has to follow the schedule, otherwise a
+        penalty term will be activated. If array-like, values can be None
+        for flexible, non-fixed flow during the certain timestep.  Used in
+        combination with :attr:`penalty_pos` and :attr:`penalty_neg`.
+    penalty_pos : numeric (sequence or scalar)
```

Comment:
Should there be  a high default value of something like 10e9 force the schedule kind of by default?

Reply:
I was also thinking about adding a default value. It's actually more user friendly. I think it depends on what kind of users are working with oemof. Do you think it will be necessary? I can easily add one :-)